src/main/java/uk/gov/hmcts/reform/pip/data/management/service/helpers/NonStrategicListFormatter.java

```diff
@@ -148,7 +152,15 @@ public final class NonStrategicListFormatter {
         Map.entry(SENIOR_COURTS_COSTS_OFFICE_DAILY_CAUSE_LIST,
                   Map.of(TIME, NonStrategicFieldFormattingHelper::formatTimeField)),
         Map.entry(MAYOR_AND_CITY_CIVIL_DAILY_CAUSE_LIST,
-                  Map.of(TIME, NonStrategicFieldFormattingHelper::formatTimeField))
+                  Map.of(TIME, NonStrategicFieldFormattingHelper::formatTimeField)),
+        Map.entry(BIRMINGHAM_ADMINISTRATIVE_COURT_DAILY_CAUSE_LIST,
+                  Map.of(HEARING_TIME, NonStrategicFieldFormattingHelper::formatTimeField)),
```

Comment:
You should be formatting time and not hearingTime

Reply:
Good spot, don't know how did I make this mistake :)